### PR TITLE
coverity 1382799, 1382796: Unchecked return value

### PR DESCRIPTION
**Pull request by bryancall**

### CODE REVIEW

**Comment by bryancall** [comment]
[approve ci autest]

**Comment by persiaAziz** [comment]
working on fixing autest for this to pass

File: iocore/net/SSLSNIConfig.cc [hunk 1]
```diff
@@ -130,8 +130,18 @@ SNIConfigParams::Initialize()
   sni_filename = ats_stringdup(RecConfigReadConfigPath("proxy.config.ssl.servername.filename"));
   lua_State *L = lua_open(); /* opens Lua */
   luaL_openlibs(L);
-  luaL_loadfile(L, sni_filename);
-  lua_pcall(L, 0, 0, 0);
+  if (luaL_loadfile(L, sni_filename)) {
+    Error("Loading SNI configuration - luaL_loadfile: %s", lua_tostring(L, -1));
```
Comment (dragon512) [inline iocore/net/SSLSNIConfig.cc]: Can this be a warning??  so we can load as we did before without error-ing out if the file does not exist

Comment (bryancall) [reply iocore/net/SSLSNIConfig.cc]: Yes, I changed it to a warning.

**Comment by persiaAziz** [comment]
[approve ci autest]

**Pull request merged**